<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowModality">
   <enum>Qt::NonModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTableView" name="tableView">
      <attribute name="horizontalHeaderCascadingSectionResizes">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
       <bool>true</bool>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1000</width>
     <height>23</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionDataBase_Info"/>
    <addaction name="actionLogin"/>
    <addaction name="actionLogout"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionDataBase_Info"/>
   <addaction name="actionLogin"/>
   <addaction name="actionLogout"/>
   <addaction name="actionExit"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QDockWidget" name="dockWidget">
   <attribute name="dockWidgetArea">
    <number>1</number>
   </attribute>
   <widget class="QWidget" name="dockWidgetContents">
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="InfoWidget" name="infoWidget" native="true"/>
     </item>
    </layout>
   </widget>
  </widget>
  <action name="actionDataBase_Info">
   <property name="icon">
    <iconset resource="StudentMan.qrc">
     <normaloff>:/resources/icon/settings.png</normaloff>:/resources/icon/settings.png</iconset>
   </property>
   <property name="text">
    <string>DataBase Info</string>
   </property>
   <property name="toolTip">
    <string>DataBase Info</string>
   </property>
   <property name="statusTip">
    <string comment="Set Database Info." extracomment="Set Database Info."/>
   </property>
  </action>
  <action name="actionLogin">
   <property name="icon">
    <iconset resource="StudentMan.qrc">
     <normaloff>:/resources/icon/login.png</normaloff>:/resources/icon/login.png</iconset>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
   <property name="statusTip">
    <string comment="Get login dialog." extracomment="Get login dialog."/>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="StudentMan.qrc">
     <normaloff>:/resources/icon/exit.png</normaloff>:/resources/icon/exit.png</iconset>
   </property>
   <property name="text">
    <string comment="Exit the system." extracomment="Exit the system.">Exit</string>
   </property>
   <property name="iconText">
    <string>Exit</string>
   </property>
   <property name="statusTip">
    <string comment="Exit the system." extracomment="Exit the system."/>
   </property>
  </action>
  <action name="actionLogout">
   <property name="icon">
    <iconset resource="StudentMan.qrc">
     <normaloff>:/resources/icon/logout.png</normaloff>:/resources/icon/logout.png</iconset>
   </property>
   <property name="text">
    <string>Logout</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>InfoWidget</class>
   <extends>QWidget</extends>
   <header>infowidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="StudentMan.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actionExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>499</x>
     <y>324</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
